packages/element/package.json

```diff
@@ -33,8 +33,8 @@
 		"@types/react-dom": "^16.9.0",
 		"@wordpress/escape-html": "file:../escape-html",
 		"lodash": "^4.17.21",
-		"react": "^17.0.1",
-		"react-dom": "^17.0.1"
+		"react": "17.0.2",
```

Comment:
Let’s keep ranges for React versions so it doesn’t lock 3rd party projects to this particular one.

Reply:
@gziolo thank you for the great callout. I reinstated the version ranges in https://github.com/WordPress/gutenberg/pull/36328/commits/5157dec43560da3b54faf1089ccdefee1af6100f.